sys-process/atop/atop-2.3.0-r1.ebuild

```diff
@@ -49,13 +49,13 @@ src_prepare() {
 	tc-export CC PKG_CONFIG
 	sed -i 's: root : :' atop.cronsysv || die #191926
 	# prefixify
-	sed -i "s:/\(usr\|etc\|var\):${EPREFIX}/\1:g" Makefile
+	sed -i "s:/\(usr\|etc\|var\):${EPREFIX}/\1:g" Makefile || die
 }
 
 src_install() {
 	emake DESTDIR="${D}" genericinstall
 	# useless -${PV} copies ?
-	rm -f "${ED%/}"/usr/bin/atop*-${PV}
+	rm -f "${ED%/}"/usr/bin/atop*-${PV} || die
```

Comment:
`rm -f` will never trigger `die`. Either don't add `die` or remove the `-F` option.